buku

```diff
@@ -6021,6 +6026,10 @@ POSITIONAL ARGUMENTS:
     -o, --open [...]     browse bookmarks by indices and ranges
                          open a random bookmark, if no arguments
     --oa                 browse all search results immediately
+    --default-scheme S   if scheme is missing from uri, assume S.
+                         valid values are http, https (default http)
+                         (override with environment variable
+                         BUKU_DEFAULT_SCHEME)
```

Comment:
…2 subsequent pairs of parens? :sweat_smile:

Also you may want to leave the elaborate description for the manpage (`buku.1` file in the repo). It'd also be a good idea to mention in there that this is specifically for the `--open` functionality.